### ISSUE TITLE
Support `tab-set`/`tab-item` in stories

### ISSUE THREAD

**Issue by LecrisUT** [teemtee/tmt#2167]
Using only the `description` the generated documentation (at least for RTD) gets a bit messy and hard to follow. For example: `specification/plans/discover/fmf` and things can get a bit messy there in that using `path` locally, from a git repo, from a dist-git has completely different meanings. In this case it makes sense to organize it in views, e.g. with `tab-set`/`tab-item`. The current alternative is to keep on expanding the path, but I don't know how versatile it can be for the long run.

With respect to fmf and how to make it in stories, I am not entirely sure.
- One option is to just use the sphinx commands directly in there, which allows for the most control and is already possible, at the expense of CLI readability and portability.
- Another option is to add replaceable commands which expand depending on the context if it's in cli or sphinx. E.g. `${tab-set-begin}`
- Another option is to focus on this use-case specifically and expand/allow `description` to be a dict:
  ```yaml
  description:
    before_text: This is a text field
    views:
      optionA: This is the text inside a `tab-item` with the tab title `optionA`
      optionB: This is another `tab-item`
    after_text: This is a text field
  ```

**Comment by psss** [comment]
I believe the docs structure has been significantly improved after moving the plugin content into a separate section, for example the `fmf` plugin section:

* https://tmt.readthedocs.io/en/stable/plugins/discover.html#fmf

As we're now using sections in the plugin docstring, the content seems quite nicely organized. @LecrisUT, do you still see space for substantial improvement?

**Comment by LecrisUT** [comment]
Could still be relevant but in the generated reference, similar to [this](https://scikit-build-core.readthedocs.io/en/latest/reference/configs.html) where it shows you how to interact via `pyproject.toml`, cli or env-var. I would say it can be a follow-up to https://github.com/teemtee/tmt/issues/2488

**Comment by happz** [comment]
Exactly these tabs would be nice for configuration keys, fmf/CLI/environment, e.g. https://tmt.readthedocs.io/en/stable/plugins/provision.html#configuration.

Edit: even without #2488...

**Comment by thrix** [comment]
Example for tabbed experience (other project, other docs):

https://thrix.github.io/nix-toolbox/getting-started/#__tabbed_1_1
